src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -183,12 +183,12 @@ public DownloadInstance datafile(@PathParam("fileId") Long fileId, @QueryParam("
         DataFile df = dataFileService.find(fileId);
         GuestbookResponse gbr = null;    
         
-        /*
+        
         if (gbrecs == null && df.isReleased()){
-            //commenting out for 4.6 SEK
-           // gbr = guestbookResponseService.initDefaultGuestbookResponse(df.getOwner(), df, session);
+            // commenting out for 4.6 SEK - ??
```

Comment:
I mentioned to @sekmiller this morning that if he's ok with it, we should simply remove this "commenting out" comment.

Reply:
I changed the comment to explain what we're doing here - that is writing a default gb response if the response hasn't been written via the interface